### ISSUE TITLE
Efficient SAM has no response to negative point prompt

### ISSUE THREAD

**Issue by summelon** [yformer/EfficientSAM#17]
I tested the ‘S’ model on my own dataset.
I intended to use negative points to eliminate some confusing regions.
However, it appears that the model shows almost no response to the negative points.
I quickly glanced through the paper but found no relevant information.
Is this behavior of the model by design, or have I overlooked any important detail?

**Comment by balakv504** [comment]
Correct. This version of Efficient SAM was not trained using negative points as input.

**Comment by summelon** [comment]
Do you mean that there is another version which supports negative points is undergoing?
It seems to me that the efficient SAM is trained for simple (one or two positive) prompt.
But for some unseen data, I think the negative point prompt from domain knowledge would be really helpful.
